laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -16,4 +16,11 @@ object eq {
       samples.forall(s => B.eqv(f(s), g(s)) )
     }
   }
+
+  implicit def tuple2Eq[A, B](implicit A: Eq[A], B: Eq[B]): Eq[(A, B)] =
+    new Eq[(A, B)] {
+      def eqv(x: (A, B), y: (A, B)): Boolean =
+        A.eqv(x._1, y._1) && B.eqv(x._2, y._2)
+    }
+
```

Comment:
That instance is required in `tests` but defined in `laws` because the `Eq[A => B]` instance was already there. I think it should ultimately come from `algebra.std`. Guess I need to open a PR there.

I'd like to not hold up this PR by waiting until this instance appears in algebra.